plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientConfig.java

```diff
@@ -143,4 +144,17 @@ public KuduClientConfig setDynamicFilteringWaitTimeout(Duration dynamicFiltering
         this.dynamicFilteringWaitTimeout = dynamicFilteringWaitTimeout;
         return this;
     }
+
+    public boolean isDeployedInSameClustersWithTrino()
+    {
+        return deployedInSameClustersWithTrino;
+    }
+
+    @Config("kudu.deployed-in-same-clusters-with-trino")
+    @ConfigDescription("If Kudu and Trino are deployed in same clusters, replica host addresses can be calculated to support optimized local scheduling")
```

Comment:
"Assign Kudu splits to replica host if worker and kudu share the same cluster"

Reply:
We need to add this property to kudu docs.

Reply:
> We need to add this property to kudu docs.

@Praveen2112 Thank you~I have improved the codes you mentioned above, please review again